### PR TITLE
moved api for FileUpload

### PR DESCRIPTION
**Pull request by zeleniucvladislav**
Related to : https://github.com/bagrinsergiu/blox-editor/issues/22182

### CODE REVIEW

File: editor/editor/editor.php [hunk 1]
```diff
@@ -1119,7 +1122,8 @@ public function getApiActions( $config = [], $context = null ) {
 			'symbolUpdate'               => $pref . Brizy_Admin_Symbols_Api::UPDATE_ACTION,
 			'symbolDelete'               => $pref . Brizy_Admin_Symbols_Api::DELETE_ACTION,
 			'symbolList'                 => $pref . Brizy_Admin_Symbols_Api::LIST_ACTION,
-			'mediaResizeUrl'             => home_url()
+			'mediaResizeUrl'             => home_url(),
```
Comment (alecszaharia) [inline editor/editor/editor.php:1125]: Delete both lines! This is the list of api actions.. not sure why you put this here.

File: public/editor-client/src/index.ts [hunk 2]
```diff
@@ -13,6 +14,10 @@ const api = {
     addMedia,
     mediaResizeUrl: config.actions.mediaResizeUrl
   },
+  customFile: {
+    addFile,
+    fileUrl: config.actions.fileUrl
```
Comment (alecszaharia) [inline public/editor-client/src/index.ts:19]: Use config.api.media.mediaResizeUrl as you will delete the keys from actions.

File: editor/editor/editor.php [hunk 3]
```diff
@@ -245,6 +245,9 @@ private function getCompilerRequiredFields( $config, $context ) {
 		$config['api'] = [
 			'media' => [
 				'mediaResizeUrl' => home_url()
+			],
+			'customFile' => [
+				'customFile' => home_url()
```
Comment (alecszaharia) [inline editor/editor/editor.php:250]: Raname customFile key to something else.. like customFileUrl..

File: public/editor-client/src/index.ts (hunk 2 above)
Comment (alecszaharia) [inline public/editor-client/src/index.ts:19]: Use config.api.customFile.customFileUrl() as you will delete the keys from actions.

**Review by alecszaharia — CHANGES_REQUESTED** [review]

**Comment by zeleniucvladislav** [comment]
How it works after changes : 
[Screencast from 03-15-2023 04:12:07 PM.webm](https://user-images.githubusercontent.com/40203375/225339263-16fcafc5-0689-48c8-9cd2-032f5c42efa4.webm)

**Pull request merged**